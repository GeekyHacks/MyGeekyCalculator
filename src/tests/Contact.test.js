import React from 'react';
import { render } from '@testing-library/react';
import Contact from '../components/Contact';

test('renders Contact correctly', () => {
  const { container } = render(<Contact />);
  expect(container).toMatchSnapshot();
});
