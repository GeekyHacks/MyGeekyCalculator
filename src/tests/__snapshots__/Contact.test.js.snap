// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Contact correctly 1`] = `
<div>
  <div
    class="ContactComponent"
  >
    <h2>
      Contact Information
    </h2>
    <h3>
      Reach out to us whenever you have any question or wanna say Hello!
    </h3>
    <ul>
      <li>
        Author:geekyhacks22@gmail.com
      </li>
      <li>
        Phone:+967777759022
      </li>
      <li>
        Adress:Zaid Street, Sanaa, Yemen
      </li>
    </ul>
  </div>
</div>
`;

exports[`renders Footer correctly 1`] = `
<div>
  <div>
    <footer>
      <h3
        class="copyRights"
      >
        Copy rights reserved to
        <a>
           GeekyHacks
        </a>
      </h3>
    </footer>
  </div>
</div>
`;
